### PR TITLE
Swagger PRs - 3 - Augmenting existing endpoints with swagdoc and correcting them

### PR DESCRIPTION
**Pull request by aronchick**

### CODE REVIEW

File: pkg/models/node_info.go [hunk 1]
```diff
@@ -63,28 +55,57 @@ func (p mergeProvider) GetLabels(ctx context.Context) map[string]string {
...
+// NodeInfo godoc
+//
+// @ID			NodeInfo
+// @Summary		Information about the node.
+// @Description	Information about the node.
+// @Tags		Ops
+// @Produce		json
+// @Success	200	{object}	NodeInfo
+// @Failure	500	{object}	string
+// @Router		/api/v1/agent/node [get]
```
Comment (simonwo) [inline pkg/models/node_info.go:79]: I made a comment in a previous PR that we shouldn't add documentation about routes on the data structures, but on the route handler itself.

**Review by simonwo — APPROVED** [review]
Apart from the comments, looks otherwise fine and valuable!

File: main.go [hunk 2]
```diff
@@ -16,6 +16,57 @@ import (
...
+//// 	@tokenUrl								https://example.com/oauth/token
+//// 	@scope.read								Grants read access
+//// 	@scope.write							Grants write access
+//// 	@scope.admin							Grants read and write access to administrative information
+//
+//// 	@securitydefinitions.oauth2.accessCode	OAuth2AccessCode
+//// 	@tokenUrl								https://example.com/oauth/token
+//// 	@authorizationUrl						https://example.com/oauth/authorize
+//// 	@scope.admin							Grants read and write access to administrative information
+
```
[the remark this answers is not in the document]
Comment (aronchick) [reply main.go:69]: Yep - this is commented out (4 '/'s) :)

File: pkg/models/node_info.go (hunk 1 above)
Comment (aronchick) [reply pkg/models/node_info.go:79]: dropped!

**Pull request merged**